### ISSUE TITLE
VoiceOver + TalkBack - issues accessing content underneath sheet, collapsing shield, interacting with content on sheet

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#432]
When the status sheet is expanded, content underneath the sheet is still available/focusable via swipe/touch navigation. Mobile screen readers announce this content as if it were a part of the visible Sheet content. 

When the screen is expanded screen readers users cannot collapse the sheet. 

There are some wonky behaviours where screen reader users can open the change your province nav element with an expanded shield however it happens from double tapping on any area of that sheet. Users cannot access the other elements on the sheet like information, change language etc. 

**Merging linked issues:**

**Content underneath sheet is available** 
https://github.com/cds-snc/covid-shield-mobile/issues/391

**Cannot close the shield when VO is on #186**
https://github.com/cds-snc/covid-shield-mobile/issues/186

**Unable to open menu items with VoiceOver on #197**
https://github.com/cds-snc/covid-shield-mobile/issues/197

**Cannot scan nav items in "tap for menu" when VO is on #185**
https://github.com/cds-snc/covid-shield-mobile/issues/185

**Content not announced with VoiceOver and some announced from screen behind #269**
https://github.com/cds-snc/covid-shield-mobile/issues/269

**Page isn't read properly using VO in iOS #292**
https://github.com/cds-snc/covid-shield-mobile/issues/292

**Page not read properly by TalkBalk [Android] #247**
https://github.com/cds-snc/covid-shield-mobile/issues/247

**Comment by JuliannaR** [comment]
@timarney @henrytao-me I merged these issues together as they are being worked on in a single PR.

**Comment by henrytao-me** [comment]
I am on it. 🙇
